src/main.rs

```diff
@@ -11,7 +11,14 @@ TO DO"
 
 fn main() {
     let args: Vec<String> = env::args().collect();
-    let db_connection = DBConnection::init_db(&args[1]);
+    let db_file_name = &args.get(1);
```

Comment:
In general isn't safe to use indexing in vectors (e.g`var[index]`) since could be `out of bounds`. You can use the `get` method that return an `Option`.